.github/workflows/markdownlint.yml

```
@@ -0,0 +1,32 @@
+name: Markdownlint
+
+on:
+  push:
+    paths:
```

```suggestion
on:
  push:
    branches:
      - "master"
    paths:
```

I am suggesting this change because without this limitation for triggering the workflow only on push to `master` Ii have seen duplicate GHA workflow executions before.

With this change the configuration would be similar to this other GHA that we use:
https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/.github/workflows/main.yml

---

If you limit this to only pushes to the default branch, you won't get checks on PR branches ahead of time, which leads to churn after opening a PR. The filter can also be useful for jobs that really depend on paths in a particular branch.

---

Ok. Then let's try it as is. 
I will resolve this thread and merge the PR.